test_project/settings.py

```diff
@@ -0,0 +1,58 @@
+import os
+
+import dj_database_url
+
+
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+DEBUG = True
+ALLOWED_HOSTS = []
+ROOT_URLCONF = 'pagination.tests.urls'
+STATIC_URL = '/static/'
+
+SECRET_KEY = 'u!%l9f3x5yjza&u-r1cxsg=yhmq7)ng(9-nz-$r!v)ml8$htpu'
```

Comment:
This doesn't need to be a production-ready value. We tend to use something like `'not-for-production'`.